RubberduckTests/Inspections/ImplicitActiveSheetReferenceInspectionTests.cs

```diff
@@ -46,22 +45,55 @@ End Sub
             var excelDeclaration = new ProjectDeclaration(new QualifiedMemberName(new QualifiedModuleName("Excel",
                     "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "Excel"), "Excel"), "Excel", true);
 
-            var listColumnDeclaration = new ClassModuleDeclaration(new QualifiedMemberName(
+            var globalDeclaration = new ClassModuleDeclaration(new QualifiedMemberName(
+                new QualifiedModuleName("Excel",
+                    "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "_Global"),
+                "_Global"), excelDeclaration, "_Global", true, null, null);
+
+            var globalCoClassDeclarationAttributes = new Attributes();
+            globalCoClassDeclarationAttributes.AddPredeclaredIdTypeAttribute();
+            globalCoClassDeclarationAttributes.AddGlobalClassAttribute();
+
+            var globalCoClassDeclaration = new ClassModuleDeclaration(new QualifiedMemberName(
+                new QualifiedModuleName("Excel",
+                    "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "Global"),
+                "Global"), excelDeclaration, "Global", true, null, globalCoClassDeclarationAttributes);
+
+            globalDeclaration.AddSubtype(globalCoClassDeclaration);
+            globalCoClassDeclaration.AddSupertype(globalDeclaration);
+            globalCoClassDeclaration.AddSupertype("_Global");
+
+            var rangeClassModuleDeclaration = new ClassModuleDeclaration(new QualifiedMemberName(
+                new QualifiedModuleName("Excel",
+                    "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "Range"),
+                "Range"), excelDeclaration, "Range", true, new List<IAnnotation>(), new Attributes());
+
+            var rangeDeclaration = new PropertyGetDeclaration(new QualifiedMemberName(
+                new QualifiedModuleName("Excel",
+                    "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "_Global"), "Range"),
+                globalDeclaration, globalDeclaration, "Range", null, null, Accessibility.Global, null, Selection.Home,
+                false, true, new List<IAnnotation>(), new Attributes());
+
+            var firstParamDeclaration = new ParameterDeclaration(new QualifiedMemberName(
+                new QualifiedModuleName("Excel",
+                    "C:\\Program Files\\Microsoft Office\\Root\\Office 16\\EXCEL.EXE", "_Global"),
+                "Cell1"), rangeDeclaration, "Variant", null, null, false, false);
+
+            var secondParamDeclaration = new ParameterDeclaration(new QualifiedMemberName(
```

Comment:
That's some serious test setup code, that could be worth extracting to some test-helper class and reused in other tests. No?